stable/kapacitor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kapacitor
-version: 1.1.3
+version: 1.1.4
```

Comment:
Add the parameter `existingSecret ` as the backward compatibility feature. I think we should bump the middle version would make more sense here.